[2024-09-22T16:28:45.816+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-22T16:28:45.823+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_processing.store_user manual__2024-09-22T16:28:40.273444+00:00 [queued]>
[2024-09-22T16:28:45.826+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_processing.store_user manual__2024-09-22T16:28:40.273444+00:00 [queued]>
[2024-09-22T16:28:45.827+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-22T16:28:45.832+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): store_user> on 2024-09-22 16:28:40.273444+00:00
[2024-09-22T16:28:45.835+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=8439) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-22T16:28:45.836+0000] {standard_task_runner.py:72} INFO - Started process 8441 to run task
[2024-09-22T16:28:45.837+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'manual__2024-09-22T16:28:40.273444+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmphnv6wzc2']
[2024-09-22T16:28:45.837+0000] {standard_task_runner.py:105} INFO - Job 86: Subtask store_user
[2024-09-22T16:28:45.857+0000] {task_command.py:467} INFO - Running <TaskInstance: user_processing.store_user manual__2024-09-22T16:28:40.273444+00:00 [running]> on host 0bbd6d2129ab
[2024-09-22T16:28:45.893+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='user_processing' AIRFLOW_CTX_TASK_ID='store_user' AIRFLOW_CTX_EXECUTION_DATE='2024-09-22T16:28:40.273444+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-22T16:28:40.273444+00:00'
[2024-09-22T16:28:45.894+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-22T16:28:45.894+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-22T16:28:45.894+0000] {logging_mixin.py:190} INFO - Current task name:store_user state:running start_date:2024-09-22 16:28:45.823692+00:00
[2024-09-22T16:28:45.894+0000] {logging_mixin.py:190} INFO - Dag name:user_processing and current dag run status:running
[2024-09-22T16:28:45.894+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-22T16:28:45.895+0000] {postgres.py:191} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',' , filename: /tmp/processed_user.csv
[2024-09-22T16:28:45.899+0000] {base.py:84} INFO - Retrieving connection 'postgres'
[2024-09-22T16:28:45.901+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-22T16:28:45.905+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-22T16:28:45.906+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, run_id=manual__2024-09-22T16:28:40.273444+00:00, execution_date=20240922T162840, start_date=20240922T162845, end_date=20240922T162845
[2024-09-22T16:28:45.909+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-09-22T16:28:45.909+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-09-22T16:28:45.909+0000] {logging_mixin.py:190} INFO - Dag name:user_processing queued_at:2024-09-22 16:28:40.283708+00:00
[2024-09-22T16:28:45.910+0000] {logging_mixin.py:190} INFO - Task hostname:0bbd6d2129ab operator:PythonOperator
[2024-09-22T16:28:45.938+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-22T16:28:45.948+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-22T16:28:45.948+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
